### PR TITLE
Improve summarize.sh, move long comments to Wiki, rename package

### PR DESCRIPTION
**Pull request by korotkevics**
* Fix public API count in summarize.sh
* Introduce package-private API count in summarize.sh
* Move long comments onto Wiki
* Rename package

### CODE REVIEW

File: shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Author.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package pl.korotkevics.ships.shared.infra.communication.api.message;
+
```
Comment (rivalez) [inline shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Author.java]: missing java doc

File: shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Author.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+public enum Author {
+  AUTO, CLIENT, SERVER
+}
```
Comment (rivalez) [inline shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Author.java]: new line

File: shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Header.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package com.epam.ships.shared.infra.communication.api.message;
+package pl.korotkevics.ships.shared.infra.communication.api.message;
 
 public enum Header {
```
Comment (rivalez) [inline shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Header.java]: java doc

File: shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Status.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package pl.korotkevics.ships.shared.infra.communication.api.message;
+
+public enum Status {
```
Comment (rivalez) [inline shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Status.java]: java doc

File: shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Author.java (hunk 1 above)
Comment (korotkevics) [reply shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Author.java]: Thx, I'll fix it!

File: shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Author.java (hunk 2 above)
Comment (korotkevics) [reply shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Author.java]: Thx, I'll fix it!

File: shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Header.java (hunk 3 above)
Comment (korotkevics) [reply shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Header.java]: Thx, I'll fix it!

File: shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Status.java (hunk 4 above)
Comment (korotkevics) [reply shared/src/main/java/pl/korotkevics/ships/shared/infra/communication/api/message/Status.java]: Thx, I'll fix it!

**Pull request merged**